deps/chakrashim/src/v8message.cc

```diff
@@ -28,8 +28,18 @@ Local<String> Message::Get() const {
 }
 
 MaybeLocal<String> Message::GetSourceLine(Local<Context> context) const {
-  // CHAKRA-TODO: Figure out how to transmit this info...?
-  return Local<String>();
+  JsPropertyIdRef source = jsrt::IsolateShim::GetCurrent()
+    ->GetCachedPropertyIdRef(jsrt::CachedPropertyIdRef::source);
```

Comment:
nit:  Elsewhere in the node code they use double-indent for continuations from the previous line to help it stand out.